lobby/src/main/java/games/strategy/engine/lobby/server/LobbyRunner.java

```diff
@@ -18,16 +19,17 @@ private LobbyRunner() {}
    * Entry point for running a new lobby server. The lobby server runs until the process is killed or the lobby server
    * is shut down via administrative command.
    */
-  public static void main(final String[] args) {
-    try {
-      final LobbyPropertyReader lobbyPropertyReader =
-          new LobbyPropertyReader(new FilePropertyReader("config/lobby/lobby.properties"));
-      log.info("Starting lobby on port " + lobbyPropertyReader.getPort());
-      LobbyServer.start(lobbyPropertyReader);
-      log.info("Lobby started");
-    } catch (final Exception e) {
-      log.log(Level.SEVERE, "Failed to start lobby", e);
-      ExitStatus.FAILURE.exit();
+  public static void main(final String[] args) throws IOException {
```

Comment:
Since `main()` is only called by the JVM, an accurate exception specification isn't really important.  `throws Exception` would be sufficient and also reduce noise a bit (similar to the convention we follow for test methods invoked exclusively by JUnit).

Reply:
I don't have a particularly strong opinion on this, my thinking was it's generally not good practice for main to throw exceptions, potential poor error handling, for test methods IMO the exception type is used more to reduce coupling than anything else.

I can update this to be 'Exception' type.

Reply:
a2888fe462508a136a2ec630ee37f765b3ea7cd4 updated the throws type.